src/test/java/io/phasetwo/service/resource/OrganizationResourceTest.java

```diff
@@ -140,13 +137,38 @@ void testAddGetUpdateDeleteOrg() throws Exception {
     assertThat(response.getStatusCode(), is(Status.NOT_FOUND.getStatusCode()));
 
     // get list
-    response = getRequest();
+    organizations = getListOfOrganizations(keycloak);
+    assertThat(organizations.size(), is(0));
+  }
 
-    assertThat(response.getStatusCode(), is(Status.OK.getStatusCode()));
-    organizations =
-        objectMapper().readValue(response.getBody().asString(), new TypeReference<>() {});
+  @Test
+  void testDeleteAllOrganizations() throws Exception {
+    final int numberOfOrgs = 3;
```

Comment:
Since the use case here is *many* organizations, let's make a test that has 1000's of organizations, and has an IdP associated with them.